vision/google/cloud/vision/_gax.py

```diff
@@ -30,24 +30,25 @@ def __init__(self, client=None):
         self._client = client
         self._annotator_client = image_annotator_client.ImageAnnotatorClient()
 
-    def annotate(self, image, features):
+    def annotate(self, images):
```

Comment:
In this case `images` is more like a tuple of pairs.  e.g. `((image, [features]),)`

I'm thinking of following this up with a PR that attaches the feature types to the `Image` instance itself.